apps/platform/pages/projects/[slug]/branches.tsx

```diff
@@ -56,6 +59,10 @@ export const BranchesPage = ({
 }: Props) => {
   const [isOpen, setIsOpen] = useState(false);
   const [isPrModalOpen, setIsPrModalOpen] = useState(false);
+  // Keep track of deletedAt status for each row
+  const [deletedRows, setDeletedRows] = useState<{ [id: string]: boolean }>({});
```

Comment:
Do we really need to track `deleted branches` ?
Just refreshing the props will do the job I think.

Reply:
Refreshing the page will resolve the issue. This particular state is used to temporarily hold the deleted status after the user clicks on delete, and to render a restore status afterward.